src/store/user-store.ts

```diff
@@ -16,16 +19,46 @@ class UserStore {
     return !!this.user?.id;
   }
 
+  public init = async () => {
+    this.isInitLoading = true;
+    this.error = '';
+
+    try {
+      const userID = localStorage.getItem(LSKeys.USER_ID);
+      if (userID) {
+        const response = await customerService.getCustomerByID(userID);
+        runInAction(() => {
+          console.log(response);
+          this.user = response;
+        });
+      }
+    } catch (error) {
+      runInAction(() => {
+        this.error = error instanceof Error ? error.message : String(error);
```

Comment:
поле this.error выводится в форму
![alt text](image.png)
если ошибка пришла как надо, типа error instanceof Error - то все выглядит красиво. в обратном случе я бы не делал  String(error). потому что то можно получить кракозябру какую-то json-оподобную в форме. я бы просто дефолтное сообщение об ошибке на этот случай завел, типа Registration error, Login error или чет подобное


```suggestion
        this.error = error instanceof Error ? error.message : messages.registerError;
```